README.md

```diff
@@ -276,6 +277,29 @@ See the [jcasc](src/test/resources/io/jenkins/plugins/opentelemetry/jcasc) folde
 For more details see the configuration as code plugin documentation:
 <https://github.com/jenkinsci/configuration-as-code-plugin#getting-started>
 
+## Advanced features
+
+You can now instrument your [Jenkins shared library](https://www.jenkins.io/doc/book/pipeline/shared-libraries/) or
+your Jenkins pipelines with the below step:
+
+* createSpan(String name, Map<String, String> attributes)
+
+For instance, let's say you would like to create a span for the docker login details
+
+```groovy
+pipeline {
+  ...
+      createSpan(name: 'my-docker-login',
+                 attributes: ['net.peer.name': 'my.docker.registry',
```

Comment:
Can you confirm that the `attributes` are exclusively associated with the created span, not associated with children spans?

Reply:
I've just added a couple of assertions in the UTs:
- https://github.com/jenkinsci/opentelemetry-plugin/pull/84/commits/16f3a6e9a0501ad4cc335f1e3080b7ba80ae5377